tests/acceptance/standard/test_group_avatar.py

```diff
@@ -0,0 +1,292 @@
+import os
+import pytest
+import tempfile
+import shutil
+from tests.acceptance import (
+    run_gitlabform,
+)
+
+
+class TestGroupAvatar:
+    def setup_method(self):
+        # Use gitlabform logo
+        self.project_root = os.path.abspath(os.path.join(os.path.dirname(__file__), "../../.."))
+        self.test_image_path = os.path.join(self.project_root, "docs/images/gitlabform-logo.png")
+
+        # Check if the file exists
+        assert os.path.exists(self.test_image_path), f"Test image not found at {self.test_image_path}"
+
+    def test__group_avatar_set_absolute_path(self, group):
+        # Test setting a group avatar with absolute path
+        config = f"""
+        projects_and_groups:
+          {group.full_path}/*:
+            group_settings:
+              avatar: "{self.test_image_path}"
+        """
+        run_gitlabform(config, group)
+
+        # Refresh group data
+        group = group.manager.get(group.id)
+
+        # Verify avatar is set
+        assert group.avatar_url is not None
+
+    def test__group_avatar_paths_comprehensive(self, group):
+        """Test both absolute and relative paths comprehensively"""
+
+        # Test 1: Absolute path (already tested above, but included for completeness)
+        config_absolute = f"""
+        projects_and_groups:
+          {group.full_path}/*:
+            group_settings:
+              avatar: "{self.test_image_path}"
+        """
+        run_gitlabform(config_absolute, group)
+
+        group = group.manager.get(group.id)
+        assert group.avatar_url is not None
+
+        # Test 2: Simple relative path
+        original_cwd = os.getcwd()
+        temp_dir = tempfile.mkdtemp()
+
+        try:
+            os.chdir(temp_dir)
+
+            # Copy image to temp directory
+            temp_image_name = "relative_test_avatar.png"
+            shutil.copy2(self.test_image_path, temp_image_name)
```

Comment:
Double-checking the logic here. Are we actually testing "relative path" usage? Based on the documentation on `tempfile.mkdtemp()`, it will return you the absolute path of the temp directory.

https://docs.python.org/3/library/tempfile.html#tempfile.mkdtemp

Please let me know if I'm looking at the wrong documentation.